src/lib/eina/eina_main.c

```diff
@@ -27,7 +27,9 @@
 # ifdef HAVE_SYS_TYPES_H
 #  include <sys/types.h>
 # endif
-# include <unistd.h>
+# ifndef _WIN32
+#  include <unistd.h>
```

Comment:
Whouldn't this change hit cygwin builds?

Reply:
Indeed, didn't realize that. So I should change to `_MSC_VER`, right?

Reply:
Ideally, we should use a combination of `_WIN32` and `__CYGWIN__` macros, but I think `_MSC_VER` does the job.